default.nix

```diff
@@ -775,7 +775,7 @@ rec {
     sha256 = "sha256-debC8ZpbIjgpEeISCISU0EVySJvf+WsUkUaLuJ526wA=";
   };
 
-  shell = stdenv.mkDerivation {
+  shell = nixpkgs.mkShell {
```

Comment:
@nomeata this was in my worktree (I've adopted it from a reddit post) and snuck in with `-a`. I guess I'll leave it in as it conveys the intent better.